otelcollector/configmapparser/default-prom-configs/clusterautoscalerDefault.yml

```diff
@@ -0,0 +1,28 @@
+  scrape_configs:
+  - job_name: cluster-autoscaler
+    scrape_interval: $$SCRAPE_INTERVAL$$
+    follow_redirects: false
```

Comment:
would these files be replaced by the new configmap?

Reply:
Yes, I'm currently working on copying these over into my branch and creating an image based on that. I will share a PR link when its ready.